<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>816</width>
    <height>469</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>50</y>
     <width>381</width>
     <height>241</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="idLabel">
      <property name="text">
       <string>ID:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="idLineEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="nameLabel">
      <property name="text">
       <string>Name:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="nameLineEdit"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="priceLabel">
      <property name="text">
       <string>Price:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="priceLineEdit"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="expirationLabel">
      <property name="text">
       <string>Expiration Date:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="expirationLineEdit"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="countLabel">
      <property name="text">
       <string>Count:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="countLineEdit"/>
    </item>
    <item row="5" column="0" colspan="2">
     <layout class="QHBoxLayout" name="buttonLayout">
      <item>
       <widget class="QPushButton" name="okButton">
        <property name="text">
         <string>OK</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cancelButton">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
